pages/api/csv-export.js

```diff
@@ -1,5 +1,6 @@
 import fire from '../../config/firebaseAdminConfig';
-const ObjectsToCsv = require('objects-to-csv');
+import ObjectsToCsv from 'objects-to-csv';
+import _ from 'lodash';
```

Comment:
Could be worth importing the individual methods used from `lodash` rather than the whole lib.